modules/ivc_champva/app/services/ivc_champva/attachments.rb

```diff
@@ -5,7 +5,7 @@ module Attachments
     attr_accessor :form_id, :uuid, :data
 
     def handle_attachments(file_path)
-      file_path_uuid = file_path.gsub("#{form_id}-tmp", "#{uuid}_#{form_id}-tmp")
+      file_path_uuid = file_path
       File.rename(file_path, file_path_uuid)
```

Comment:
this is a drive by comment but if file_path and file_path_uuid are the same, what is the File.rename call doing?

Reply:
fixed